cosmo_tester/test_suites/cluster/conftest.py

```diff
@@ -177,6 +188,13 @@ def _get_hosts(ssh_key, module_tmpdir, test_config, logger, request,
         for node in hosts.instances:
             node.verify_services_are_running = skip
 
+        if installer_image_name:
+            # The cluster images distro is rhel by default, so let's keep it
```

Comment:
It's centos if you specify to use centos in the test config, so we should support that.

Reply:
Thanks, will add a centos support.